tests/testthat/test-markdown.R

```diff
@@ -36,3 +36,10 @@ test_that("markdown_body() captures title", {
   expect_false(grepl("Title", html))
 })
 
+test_that("markdown_body() can parse UTF-8", {
```

Comment:
I think you could simplify this test to something like:

```R
test_that("markdown_text_*() handles UTF-8 correctly", {
  expect_equal(markdown_text_block("\u00f8"), "<p>\u00f8</p>")
  expect_equal(markdown_text_inline("\u00f8"), "\u00f8")
})
```

Otherwise, PR looks good and thanks for working on this!